### ISSUE TITLE
Question: Conflict resolution

### ISSUE THREAD

**Issue by MindTwister** [ejcx/passgo#25]
How does passgo handle conflict resolution between remote and local?

I want to sync  between my work and home computer, but sometimes I forget to push at home before inserting a new entry at work, since the JSON isn't prettyprinted I'm guessing either the work data or the home data will be lost to the ether.

**Comment by ejcx** [comment]
I actually do not handle conflicts and conflict resolution in passgo.

Any number of things can go wrong and I didn't want people to lose their passwords and make a 'solution' that doesn't actually fix an issue.

My recommendation is `cd ~/.passgo` and resolving the issue manually.

The obvious fix to this is to separate out each password entry to be in it's own file so git will complain less, but that makes the integrity checking of each site a lot lot lot more involved.

**Comment by ejcx** [comment]
@MindTwister did you ever resolve the conflicts? How did it go? Is there anything that you think I should be doing to help?

**Comment by MindTwister** [comment]
@ejcx I never got around to look at it, my current plan will be to prettyprint the files, commit, push and then on the other machine prettyprint, commit and pull, I'll let you know how it works out when I've tried it.

**Comment by ejcx** [comment]
Another PR request is prettyprinting files. I definitely see the value in pretty printing for merge resolves. I'll try to review it.

**Comment by MindTwister** [comment]
I just tried a manual merge, this has potential to go horribly wrong, the data is just too similar for any merge tool. If I took the merge at face value it would simply have renamed some sites instead of actually adding/removing sites.

Maybe if the data was prettyprinted from the get-go it would be possible for git to make any sense of it, but at the very least the prettyprinted PubKey in sites.json should be in one line.

What I ended up doing was:

1. Get the two sites.json files
2. Prettyprint the json using `python -m json.tool`
3. Join the "PubKey" array for each site on one line
4. Manually/visually perform the merge
5. Replace sites.json with the merged file

It seems to have worked just fine, I'll post here if any problems arises.
